libs/langchain_v1/tests/unit_tests/agents/test_react_agent.py

```diff
@@ -52,6 +56,26 @@
 pytestmark = pytest.mark.anyio
 
 
+@overload
+def _create_state_graph(
+    state_schema: type[StateT],
+    output_schema: None = None,
+) -> StateGraph[StateT, None, StateT, StateT]: ...
```

Comment:
This helps typing `StateGraph` when `output_schema` is not provided.
Maybe there could be a static method in `StateGraph` ?